aws/ec2metadata/service_test.go

```diff
@@ -92,7 +92,7 @@ func TestClientDisableIMDS(t *testing.T) {
 	svc := ec2metadata.New(unit.Session, &aws.Config{
 		LogLevel: aws.LogLevel(aws.LogDebugWithHTTPBody),
 	})
-	resp, err := svc.Region()
+	resp, err := svc.GetUserData()
```

Comment:
Shouldn't this still be `Region()`?

Reply:
It shouldn't matter as we do not test api operations here. This is a client test. We test api operations in api_test.go

Reply:
Why make the change then in this case?

Reply:
We updated the GetRegion method to use Instance identity document. The mock server can be changed to serve a Instance identity document instead to work with GetRegion in this test case. I can make the change but it's seemed like an unnecessary addition in this case. As we do not actually test GetRegion behavior here.